openpype/settings/entities/schemas/system_schema/schema_general.json

```
@@ -130,6 +130,10 @@
             "label": "OpenPype deployment control",
             "collapsible": false,
```

There is too much now
```suggestion
            "collapsible": true,
```

---

Damn, it is just a wrapped so it won't show overriden state... Maybe not.